ecommerce/api.py

```diff
@@ -854,7 +859,9 @@ def validate_basket_for_checkout(basket):
     # All run ids must be purchasable and enrollable by user
     for run in runs:
         if not run.is_unexpired:
-            raise ValidationError({"runs": f"Run {run.id} is expired"})
+            raise ValidationError(
+                {"runs": f"Course run readable_id ({run.courseware_id}) is expired"}
```

Comment:
This message is shown to users, who do not know what a "course run" is and certainly not a "readable_id". I know that a lot of existing validation messages like this mention "runs" and do not seem to be aimed at users, but it's a good time to start changing that. Please change to this:

```suggestion
                {"runs": f"Course '{selection.run.title}' is not accepting enrollments."}
```

Reply:
I did exactly what was mentioned on the ticket, but changes it as you suggested.